packages/framework/src/Controller/Admin/PaymentController.php

```diff
@@ -71,7 +71,9 @@ public function newAction(Request $request)
     {
         $paymentEditData = $this->paymentEditDataFactory->createDefault();
 
-        $form = $this->createForm(PaymentEditFormType::class, $paymentEditData);
+        $form = $this->createForm(PaymentEditFormType::class, $paymentEditData, [
```

Comment:
This is actually important for usage of ImageuplouadType. I think that it is important to have these changes in same commit

Reply:
Mind to tell me why?

Reply:
Because in this commit i created ImageUploadType which works wich expects to have some entity which has images related to it, so i send this object in PaymentController, I actually did the same with `TransportController` so i dont really understand why it is problem with `paymentControler`